ocaml/file_formats/cmx_format.mli

```diff
@@ -33,7 +33,20 @@ type export_info =
   | Clambda of Clambda.value_approximation
   | Flambda of Export_info.t
 
-type apply_fn := int * Lambda.alloc_mode
+(* Declare machtype here to avoid depending on [Cmm]. *)
+type machtype_component = Val | Addr | Int | Float
+type machtype = machtype_component array
+
+type apply_fn := machtype list * machtype * Lambda.alloc_mode
+
+(* Symbols of function that pass certain checks for special properties. *)
```

Comment:
Looks like this section might have been accidentally pasted?

Reply:
Oops indeed, will remove that.